pycytominer/cyto_utils/DeepProfiler_processing.py

```diff
@@ -369,6 +394,8 @@ def get_single_cells(self, output=False):
         -----------
         output : bool
             If true, will output the single cell dataframe instead of setting to self attribute
+        location_columns : tuple
```

Comment:
make sure to update this once you address my comment in `load_npz_locations()`. 

I was a bit confused reading this until I read the proposed changes to this function (`load_npz_locations`).

Reply:
Resolved in https://github.com/cytomining/pycytominer/pull/218/commits/6d5eb23bcf072c7ac344ada78bbc305d6f9ece49